artisatomic/readtanakajpltdata.py

```diff
@@ -44,10 +44,19 @@ def read_levels_and_transitions(atomic_number, ion_stage, flog):
         filename = f"{filename}.zst"
     print(f"Reading Tanaka et al. Japan-Lithuania database for Z={atomic_number} ion_stage {ion_stage} from {filename}")
     with xopen(jpltpath / filename) as fin:
-        artisatomic.log_and_print(flog, fin.readline().strip())
-        artisatomic.log_and_print(flog, fin.readline().strip())
-        artisatomic.log_and_print(flog, fin.readline().strip())
-        assert fin.readline().strip() == f"# {atomic_number} {ion_stage}"
+        counter = 0
+        while True:
+            readlinein = fin.readline().strip()
+            artisatomic.log_and_print(flog, readlinein)
+            counter += 1
+            if readlinein == f"# {atomic_number} {ion_stage}":  # search for this line. Header info can be different
+                artisatomic.log_and_print(flog,
+                        f"Reading Tanaka et al. Japan-Lithuania database for Z={atomic_number}"
+                        f" ion_stage {ion_stage} from {filename}")
+                break
+            if counter > 6:
+                break
```

Comment:
I'm not sure how the 6 was determined, but this counter and while loop is just a very complicated way to write a simple for loop.
```python
counter = 0
while (True):
  print(counter)
  counter += 1
  if counter > 6:
    break
````

Is the same as:
```python
for counter in range(7):
  print(counter)
```